ethcore/src/state.rs

```diff
@@ -571,7 +573,21 @@ fn should_not_trace_call_transaction_to_builtin() {
 
 	let result = state.apply(&info, engine.deref(), &t, true).unwrap();
 
-	assert_eq!(result.trace, None);
+	assert_eq!(result.trace, Some(Trace {
```

Comment:
I turned on tracing of transactions to builtins. It simplifies querying traces by transaction hash, cause number of traces is always equal to number of transactions in given block.